act-rules-format.bs

```diff
@@ -15,86 +15,112 @@ Markup Shorthands: markdown yes
+
+The separation between atomic rules and composed rules creates a division of responsibly. Atomic rules test if web content correctly implemented a particular solution. Composed rules test if a combination of `pass` outcomes from atomic rules is sufficient for an accessibility requirement not to `fail`. Not all atomic rules have to be part of a composed rule. Only if failing that atomic rule does not directly fail an [Accessibility Requirements](#structure-accessibility-requirements). An atomic rule MAY be part of multiple composed rules.
+
+A composed rule define how the outcomes from its atomic rules are aggregated into a single outcome for each applicable test target. Note that atomic rules in a composed rule MAY have different applicability. Because of this, not every element applicable within the composed rule is tested by every atomic rule. Atomic rules MAY also be disabled during a test run due to accessibility support concerns. See [Accessibility support](#acc-support) for details.
+
+<div class="example">
+  <p>Composed rule: Header cells in HTML tables (WCAG 2 Success Criterion 1.3.1)</p>
+  <p>This rule would use results from the following atomic rules:</p>
+  <ul>
+    <li>Header indicated through implicit scope</li>
+    <li>Header indicated through the `scope` attribute</li>
+    <li>Header indicated by using the `headers` attribute</li>
+    <li>Header indicated by using ARIA labels</li>
+  </ul>
+  <p>If any one of these rules passes, the table cell can pass the composed rule.</p>
+</div>
 
 
 ACT Rule Structure {#structure}
-==============================
+===============================
 
-Rule Outline {#structure-outline}
------------------------
+Atomic Rule Outline {#structure-outline}
+----------------------------------------
 
-An ACT Rule MUST be written in plain language. It MUST consist of the following items:
+An Atomic ACT Rule MUST consist of the following items:
 
 * Descriptive title
-* [Unique identifier](#unique-identifier)
+* [Rule identifier](#rule-identifier)
 * [Rule Description](#structure-description)
 * [Accessibility Requirements](#structure-accessibility-requirements)
-* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
-* [Accessibility Support](#acc-support) (optional)
 * [Aspects Under Test](#input-aspects)
 * [Test Definition](#test-def)
+* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
+* [Accessibility Support](#acc-support) (optional)
 
-Unique Identifier {#unique-identifier}
-----------------------------------------
+Composed Rule Outline {#structure-outline}
+------------------------------------------
+
+An Composed ACT Rule MUST consist of the following items:
+
+* Descriptive title
+* [Rule identifier](#rule-identifier)
+* [Rule Description](#structure-description)
+* [Aggregated Rules](#aggregated-rules)
```

Comment:
I didn’t understand the heading  "Aggregated rules" at first read. I think my confusion comes from that it is very close to "composed rules" but means a different thing here.
It might be made more explicit by saying "list of aggregated rules", or "rule composition".

But anyways, I think this section could actually be merged with "aggregation definition" (see my comment below).